pkg/virt-launcher/virtwrap/network/dhcpv6/dhcpv6.go

```diff
@@ -123,6 +123,11 @@ func (h *DHCPv6Handler) ServeDHCPv6(conn net.PacketConn, peer net.Addr, m dhcpv6
 		response, err = dhcpv6.NewReplyFromMessage(msg, h.modifiers...)
 	}
 
+	ianaRequest := msg.Options.OneIANA()
+	ianaResponse := response.Options.OneIANA()
+	ianaResponse.IaId = ianaRequest.IaId
+	response.UpdateOption(ianaResponse)
```

Comment:
Could you please explain what this means in the commit message?

Reply:
sure, done.

Reply:
Sorry, done.